src/customHooks/useWindowDimensions.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height
+  };
+}
+
+export default function useWindowDimensions() {
+  const [windowDimensions, setWindowDimensions] = useState(getWindowDimensions());
+
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions(getWindowDimensions());
+    }
+
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+  return windowDimensions;
```

Comment:
because we don't care about the actual demensions, maybe instead of returning the values
you can return if it desktop size or mobile size.
something like : return windowDimensions.width > 780
and than its just a boolean value which is much easier to use

Reply:
not a must, just a suggestion

Reply:
sounds good! changed it.